sos/report/plugins/bcache.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2021 Ponnuvel Palaniyappan <ponnuvel.palaniyappan@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Bcache(Plugin, IndependentPlugin):
+
+    short_desc = 'Bcache statistics'
+
+    plugin_name = 'bcache'
+    profiles = ('storage', 'hardware')
+    files = '/sys/fs/bcache'
+
+    # Caution: reading /sys/fs/bcache/*/cache0/priority_stats is known
```

Comment:
Perhaps we should add this to a forbidden list, as a safeguard beyond the note here? I'm thinking what if this location for one reason or another changes and a `priority_stats` is added in an otherwise captured spot? The forbidden path glob could be fairly wide open, e.g. `/sys/fs/bcache/*/priority_stats` or similar

Reply:
Yes, forbidden list makes sense. Does glob expansion considered as access/read?

Did quick test with:
```
import glob

for name in glob.glob('/sys/fs/bcache/*/cache0/priority_stats'):
    print name
```
and it does `lstat` and `fstat` on it:

```
...
lstat("/sys/fs/bcache/register_quiet/cache0", 0x7ffdd00ee980) = -1 ENOTDIR (Not a directory)
lstat("/sys/fs/bcache/register/cache0", 0x7ffdd00ee980) = -1 ENOTDIR (Not a directory)
lstat("/sys/fs/bcache/583c9130-fa01-455f-9e96-e0b5c84b2902/cache0", {st_mode=S_IFLNK|0777, st_size=0, ...}) = 0
lstat("/sys/fs/bcache/583c9130-fa01-455f-9e96-e0b5c84b2902/cache0/priority_stats", {st_mode=S_IFREG|0444, st_size=4096, ...}) = 0
fstat(1, {st_mode=S_IFCHR|0620, st_rdev=makedev(136, 4), ...}) = 0
write(1, "/sys/fs/bcache/583c9130-fa01-455"..., 74) = 74
...
```
Just want to make sure the forbidden list doesn't end up doing what we want to avoid here.

Reply:
I wouldn't think the glob would trigger the referenced issue, stat'ing just returns the contents of the file's i-node and not the content of the file directly.

Reply:
Thanks. Addressed both comments now.